cantabular-import/dp-cantabular-filter-flex-api.yml

```diff
@@ -15,9 +15,15 @@ services:
         ports:
             - 27100:27100
         restart: unless-stopped
+        depends_on:
+            - kafka-1
+            - kafka-2
+            - kafka-3
+            - mongodb
         environment:
             BIND_ADDR:                   ":27100"
-            SERVICE_AUTH_TOKEN:          $SERVICE_AUTH_TOKEN
+            SERVICE_AUTH_TOKEN:           $SERVICE_AUTH_TOKEN
```

Comment:
Should this line be:
            SERVICE_AUTH_TOKEN:                 "testToken"

, please ask David Subiros